scripts/wsrep_sst_common.sh

```diff
@@ -373,7 +384,9 @@ wsrep_gen_secret()
     fi
 }
 
->>>>>>> wsrep_5.7.34-25.26
+# Returns the absolute path from a path to a file (with a filename)
```

Comment:
This isn't strictly a merge conflict, but this comment is incorrect: the function doesn't return the absolute path, it prints the absolute path and returns a status code.

Reply:
Fixed